lib/sejfguru_web/router.ex

```diff
@@ -10,23 +10,27 @@ defmodule SejfguruWeb.Router do
   end
 
   pipeline :browser_auth do
-    plug SejfguruWeb.AuthAccessPipeline
+    plug SejfguruWeb.AuthOptionalPipeline
+  end
+
+  pipeline :browser_auth_required do
+    plug SejfguruWeb.AuthRequiredPipeline
   end
 
   pipeline :api do
     plug :accepts, ["json"]
   end
 
   scope "/", SejfguruWeb do
-    pipe_through :browser # Use the default browser stack
+    pipe_through [:browser, :browser_auth]
```

Comment:
I think `browser_auth_optional` might be more consistent way to name this pipeline (appropriate modules have `Optional` or `Required` in their names).

Also what about `browser_optional_auth` and `browser_required_auth`? It might sound a bit more natural.

Reply:
Done!